components/pricing-new/ForCollectiveCard.js

```diff
@@ -0,0 +1,293 @@
+  },
+  'pricing.outsideFunds': {
+    id: 'pricing.outsideFunds',
+    defaultMessage: 'Outside funds',
+  },
+  'pricing.outsideFunds.description': {
+    id: 'pricing.outsideFunds.description',
+    defaultMessage:
+      "Manually credit Collective budgets with funds received outside the platform (e.g. payments you've invoiced, cash, or third party channels like a shop).",
+  },
+  'pricing.bankTransfer': {
+    id: 'pricing.bankTransfer',
+    defaultMessage: 'Bank transfer payments',
+  },
+  'pricing.bankTransfer.description': {
+    id: 'pricing.bankTransfer.description',
+    defaultMessage:
+      'Enable bank transfer payments to automatically provide financial contributors with wire instructions and a reference number for tracking.',
+  },
+  'pricing.creditCard': {
+    id: 'pricing.forCollective.creditCard',
+    defaultMessage: 'Credit card processing',
+  },
+  'pricing.creditCard.description': {
+    id: 'pricing.creditCard.description',
+    defaultMessage:
+      'Receive financial contributions to you via credit card, automatically updating your budget for transparent tracking. *Stripe fees apply',
+  },
+});
+
+const ForCollectiveCard = () => {
+  const intl = useIntl();
+
+  return (
+    <Card padding={['24px', null, '32px']} width={['288px', '636px']} borderRadius="8px" border="1px solid #DCDEE0">
+      <Flex justifyContent="center" alignItems="center" mb={['32px', '35px']}>
+        <Box width="72px" height="72px" mr="16px">
+          <Illustration src="/static/images/pricing/for-collective-illustration.png" alt="Collective Illustration" />
+        </Box>
+        <Box width={[null, '500px', '672px', null, '702px']}>
+          <H3
+            fontSize={['18px', '20px']}
+            lineHeight={['26px', '28px']}
+            color="black.900"
+            fontWeight="500"
+            letterSpacing={[null, '-0.008em']}
+          >
+            <FormattedMessage id="pricing.collectiveCard" defaultMessage="For Collectives" />
+          </H3>
+          <StyledHR my="8px" />
+          <P fontSize="14px" lineHeight="20px" color="black.800">
+            <FormattedMessage
+              id="pricing.forCollective.description"
+              defaultMessage="Bring your initiative to life in no time"
+            />
+          </P>
+        </Box>
+      </Flex>
+      <Flex flexDirection={['column', 'row']} alignItems={['flex-start', null, 'center']}>
+        <Container mr={[null, '41px', '72px']}>
+          <H5
+            fontSize={['18px', '20px']}
+            lineHeight={['26px', '28px']}
+            letterSpacing={[null, '-0.008em']}
+            color="blue.700"
+            mb="16px"
+          >
+            <FormattedMessage id="pricing.forCollective.fees.header" defaultMessage="We want to see you thrive" />
+          </H5>
+          <Box mb="16px">
+            <Flex mb={3}>
+              <FeeData>0$</FeeData>
```

Comment:
It should be part of the string and we should format it passing <FeeData> to `values` in FormattedMessage.